### PR TITLE
Fix NPM security audit warnings

### PR DESCRIPTION
**Pull request by vishesh**
Updates Gulp version and top-level `package.lock`.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -586,12 +635,12 @@
...
       "requires": {
-        "fs.realpath": "1.0.0",
-        "inflight": "1.0.6",
-        "inherits": "2.0.3",
-        "minimatch": "3.0.4",
-        "once": "1.4.0",
-        "path-is-absolute": "1.0.1"
+        "fs.realpath": "^1.0.0",
+        "inflight": "^1.0.4",
+        "inherits": "2",
```
Comment (stchang) [inline package-lock.json]: Should this have caret `^`?

File: package-lock.json [hunk 2]
```diff
@@ -674,8 +723,8 @@
       "integrity": "sha1-Sb1jMdfQLQwJvJEKEHW6gWW1bfk=",
       "dev": true,
       "requires": {
-        "once": "1.4.0",
-        "wrappy": "1.0.2"
+        "once": "^1.3.0",
+        "wrappy": "1"
```
Comment (stchang) [inline package-lock.json]: same question as above

File: package-lock.json [hunk 3]
```diff
@@ -955,7 +996,7 @@
       "integrity": "sha1-WDsap3WWHUsROsF9nFC6753Xa9E=",
       "dev": true,
       "requires": {
-        "wrappy": "1.0.2"
+        "wrappy": "1"
```
Comment (stchang) [inline package-lock.json]: same question as above

**Comment by stchang** [comment]
Dont have much experience with these config files but I reinstalled and tests seem to be passing.

**Comment by stchang** [comment]
Just had one question about maybe missing carets?

**Comment by stchang** [comment]
ah ok looks good then

**Comment by vishesh** [comment]
Oups, misread your question. You asked if we should add caret. I think overall it should be fine, as same points apply.

**Pull request merged**